### PR TITLE
Renamed the command descriptions

### PR DESCRIPTION
**Pull request by aschempp**

### CODE REVIEW

File: src/Task/Maintenance/UnlockTask.php [hunk 1]
```diff
@@ -20,7 +20,7 @@ public function getName()
      */
     public function getDescription()
     {
-        return '[Terminal42] Maintenance – disable maintenance mode';
+        return '[Maintenance] Disabling maintenance mode';
```
Comment (Toflar) [inline src/Task/Maintenance/UnlockTask.php:23]: I would name the bundle here.

File: src/Task/Symfony/AcceleratorCacheClearTask.php [hunk 2]
```diff
@@ -20,7 +20,7 @@ public function getName()
      */
     public function getDescription()
     {
-        return '[Terminal42] Clear accelerator cache';
+        return '[Symfony] Clear accelerator cache';
```
Comment (Toflar) [inline src/Task/Symfony/AcceleratorCacheClearTask.php:23]: I would name the bundle here.

**Review by Toflar — CHANGES_REQUESTED** [review]

**Review by qzminski — APPROVED** [review]

**Comment by aschempp** [comment]
ping @terminal42/pilots

**Comment by aschempp** [comment]
Merged in 18aff560c777d6915f185024187c639b248e9393

**Pull request closed without merging**